openfeature-provider/java/src/main/java/com/spotify/confidence/SwapWasmResolverApi.java

```diff
@@ -76,30 +71,15 @@ public CompletableFuture<ResolveFlagsResponse> resolveWithSticky(
     switch (response.getResolveResultCase()) {
       case SUCCESS -> {
         final var success = response.getSuccess();
-        // Store updates if present
```

Comment:
Not relevant for this PR, but how do we intend to increase visibility over possible failed store updates?